ush/parsing_model_configure_FV3.sh

```diff
@@ -49,16 +49,8 @@ nbits:                   ${nbits:-14}
 imo:                     $LONB_IMO
 jmo:                     $LATB_JMO
 output_fh:               $OUTPUT_FH
-nsout:                   $NSOUT
 iau_offset:              ${IAU_OFFSET:-0}
 EOF
 
-if [ $cpl = .true. ]; then
```

Comment:
Isn't this still needed?

Reply:
This routine was carefully updated to match the chosen p8b settings and be consistent with what is used in the ufs-weather-model benchmark p8 test for coupled and standalone runs and was discussed on the Wednesday Coupled Model tag-ups.